releasenotes/notes/civisibilitywriter-3fa0ac9af42256c2.yaml

```diff
@@ -0,0 +1,4 @@
+features:
+  - |
+    tracing: Introduces ``CIVisibilityWriter``, an ``HTTPWriter`` subclass that communicates with Datadog's CI Visibility intake endpoint.
+    An upcoming change will configure the pytest plugin to automatically use this writer.
```

Comment:
```suggestion
```
On second thought, I think we should leave this out since this doesn't really pertain to this PR. 
Furthermore, I'm not sure if this should have a changelog since we're not actually changing the public API, right? The only reason I think we should add a changelog is if we want to let users know how to use agentless reporting (to set `DD_CIVISIBILITY_AGENTLESS_ENABLED=true`), but this would be in a future PR anyway.